packages/dropdown/src/styledComponents/DropdownMenuWrapper.js

```diff
@@ -7,6 +7,9 @@ const DropdownMenuWrapper = styled.div`
   min-width: 100%;
   width: ${props => props.width};
   whitespace: no-wrap;
+  max-height: ${props => props.maxHeight}px;
```

Comment:
receive `maxHeight` as string. We can set a percentage here as well.

Reply:
Just to point out, a percentage won't work well here. The percentage would be relative to the components wrapper. Here's an example with `maxHeight='50%'`:
![image](https://user-images.githubusercontent.com/1459283/43263443-a6fb511e-90b9-11e8-8479-305dd33bbc0c.png)

We need to use a viewport unit. `maxHeight='50vh':
![image](https://user-images.githubusercontent.com/1459283/43263401-9012f146-90b9-11e8-8dc9-2de2ebf92f8f.png)